library/ui-styles/src/main/res/values-h480dp/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="ftue_auth_profile_picture_height" format="float" type="dimen">0.15</item>
+    <item name="ftue_auth_profile_picture_icon_height" format="float" type="dimen">0.05</item>
```

Comment:
Those 2 values are the same than in default resource, is it a mistake?

Reply:
ah great catch :eyes: ! they can be deleted, they were separated whilst checking with design about the different screen sizes, we chose to maintain the same _% of screen_  for small screens and allow the entire avatar to be clickable instead of making the image bigger

Reply:
https://github.com/vector-im/element-android/pull/5323/commits/1fc1935374c214c1e4fe7a9205c2cb248e7740db